dispatch/static/manager/src/js/components/fields/IntField.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import { TextInput } from '../inputs'
+
+export default class IntField extends React.Component {
+  onChange(val) {
+    val = val.replace(/[^\d]/, '')
```

Comment:
as opposed to `/[^\d]/g`?

without the global flag you can't guarantee that all non-numbers will be removed, only the first character. is that intentional?

```js
'fx123abc'.replace(/[^\d]/, '')
// 'x123abc'

'fx123abc'.replace(/[^\d]+/, '')
// '123abc'

'fx123abc'.replace(/[^\d]/g, '')
// '123'